MobileFilter/sections/general_extensions.txt

```diff
@@ -58,6 +58,8 @@ superomsk.ru#$#body { overflow: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36188
 trashbox.ru#$#.div_landing {display: block!important;}
 trashbox.ru#$##div_landing_safe.show { display: none!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/36360
+sutin0831.pixnet.net#$#.pixnet-happix-pilot__sticky-box { position: absolute!important; left: -3000px!important; }

```

Comment:
Add this property to the element too:
[data-link*="/redirect/widgets"]

sutin0831.pixnet.net#$#.pixnet-happix-pilot__sticky-box[data-link*="/redirect/widgets"] { position: absolute!important; left: -3000px!important; }